src/components/button.tsx

```diff
@@ -16,28 +17,23 @@ const CustomButton = styled.button(() => [
             padding-bottom: 5px;
         }
 
-        &:hover {
-            div {
-                ${tw`opacity-0`}
-            }
+        @media screen and (min-width: 1024px) {
+            &:hover {
+                span {
```

Comment:
we could use TW for that. Something like lg:hover: should do

Reply:
How would you go about targeting the span that way tho?